### PR TITLE
aftman 0.2.7 (new formula)

### PR DESCRIPTION
**Pull request by iivusly**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

File: Formula/aftman.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  desc "Toolchain manager for Roblox, the prodigal sequel to Foreman"
+  homepage "https://github.com/LPGhatguy/aftman"
+  url "https://github.com/LPGhatguy/aftman/archive/refs/tags/v0.2.7.tar.gz"
+  sha256 "2c4f191bfce631abff185a658b947105055da687c409e09ea80786be4c32b75e"
+  license "MIT"
+  head "https://github.com/LPGhatguy/aftman.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@1.1"
```
Comment (carlocab) [inline Formula/aftman.rb:11]: No linkage with OpenSSL on macOS. We should double-check that this isn't building a vendored OpenSSL.

File: Formula/aftman.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  sha256 "2c4f191bfce631abff185a658b947105055da687c409e09ea80786be4c32b75e"
+  license "MIT"
+  head "https://github.com/LPGhatguy/aftman.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@3"
+  end
```
Comment (carlocab) [inline Formula/aftman.rb:14]: Are we sure this doesn't use OpenSSL on macOS?

Comment (iivusly) [reply Formula/aftman.rb:14]: Output from `cargo tree` does not include the package `openssl` on macOS, whereas on Linux it does. Im assuming it has to do with builtin functionality with CoreFoundation. Both of the packages above are needed to compile the package `openssl-sys` on Linux.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request merged**